src/lib/remote-place-cache.ts

```diff
@@ -31,8 +24,24 @@ export type RemotePlace = {
 };
 
 export default class RemotePlaceCache {
-  private static cache: RemotePlaceDatastore = {};
+  private static cache: NodeCache;
```

Comment:
```suggestion
  private static cache: NodeCache = new NodeCache({
      stdTTL: this.CACHE_TTL,
      checkperiod: this.CACHE_CHECK_PERIOD 
  });
```

Can get rid of getCache() if you do this inline